packages/dds/merge-tree/src/test/testClient.ts

```diff
@@ -67,7 +67,8 @@ export class TestClient extends Client {
                 logger: client2.logger,
                 clientId: newLongClientId,
             } as IFluidDataStoreRuntime,
-            services);
+            services,
+            undefined);
```

Comment:
can you please add a comment for undefined? i.e.
undefined); // name of arg...

Reply:
I created a MockSerializer to pass here instead of passing undefined since it is a required parameter.

Reply:
**Mock** is for testing :) **Null**TelemetryLogger, **Noop**Serializer

Reply:
This is actually used for testing. Earlier, the tests worked when serializer was undefined because SnapshotChunks and SnapshotV1 has serializer as optional param.

I changed the serializer param to be required everywhere so that we don't miss anything. Removed MockSerializer from test utils. Added a TestSerializer for merge tree tests only since no one else needs it.